goguen/formal-spec/transactions.tex

```diff
@@ -264,7 +223,7 @@ \subsection{Goguen transactions}
   that may be used by all non-native scripts in the transaction.
   The budget is intended to be computed by executing the scripts off-chain prior to transaction submission.
   \item We include a hash of the the indexed redeemer structure
-    with accessor $\fun{rdmrsHash}$. The reason for including this hash is explained below.
+    with accessor $\fun{sdHash}$. The reason for including this hash is explained below.
```

Comment:
unless i missed it, I dont think the reason for this is actually explained anywhere.

I would suggest making a section about "Ensuring witness integrity" or "Including optional witness data for communication purposes" , or something like that.

Reply:
Thanks, I missed that. The previous sentence is also wrong :)